Firestore/Example/Tests/Local/FSTMutationQueueTests.mm

```diff
@@ -130,26 +138,26 @@ - (void)testLookupMutationBatch {
 
   // Searching on an empty queue should not find a non-existent batch
   self.persistence.run("testLookupMutationBatch", [&]() {
-    FSTMutationBatch *notFound = [self.mutationQueue lookupMutationBatch:42];
+    FSTMutationBatch *notFound = self.mutationQueue->LookupMutationBatch(42);
     XCTAssertNil(notFound);
 
-    NSMutableArray<FSTMutationBatch *> *batches = [self createBatches:10];
-    NSArray<FSTMutationBatch *> *removed = [self removeFirstBatches:3 inBatches:batches];
+    std::vector<FSTMutationBatch *> batches = [self createBatches:10];
+    std::vector<FSTMutationBatch *> removed = [self removeFirstBatches:3 inBatches:&batches];
 
     // After removing, a batch should not be found
-    for (NSUInteger i = 0; i < removed.count; i++) {
-      notFound = [self.mutationQueue lookupMutationBatch:removed[i].batchID];
+    for (NSUInteger i = 0; i < removed.size(); i++) {
```

Comment:
This is fine.

Consider s/NSUInteger/size_t/ (which is the proper return type of vector::size()) since we'll need to do that eventually once we finish porting to c++. If you make that change here, also make it below.

Optional.

Reply:
Done.